### DISCUSSION TITLE
code-server Support built-in browser

### DISCUSSION THREAD

**Discussion by mlb0925** [coder/code-server#6934] [Q&A]
![image](https://github.com/user-attachments/assets/c8a8a1b0-18a7-4c90-9716-006a3e8a6a3a)
think you

**Comment by code-asher** [comment]
I think this might be an upstream bug.  If you look at the ports panel, you should see a URL where you can view the port, but I am not sure if there is a way to make this work with the simple browser built-in.

**Reply by mlb0925** [reply]
The computer's VSCode supports simple browsers, but coder-server does not support simple browsers? Is there a way to support Simple Brower，think you

**Reply by code-asher** [reply]
code-server uses the web version of VS Code, which sometimes works differently from their desktop version.

if we can reproduce the bug in Codespaces, we should report it to https://github.com/microsoft/vscode

**Reply by mlb0925** [reply]
thank you
